open-craft/edx-learndot
# Backport to Ginkgo

This branch contains changes necessary to run edx-learndot under Open edX Ginkgo.

**Testing instructions**:

1. Set up a Gingko devstack. 
2. Add 'edxlearndot' to settings.INSTALLED_APPS.
3. Add a valid Learndot API key to settings.LEARNDOT_API_KEY.
4. Add a valid Learndot API sandbox URL to settings.LEARNDOT_API_BASE_URL.
5. Create a course you can pass. A new course with one gradable unit that's 100% of the grade would work.
6. Configure a CourseMapping in the Django admin, using a valid Learndot component ID and the course key from the course to be passed.
7. Enroll a learner in the course, complete the gradable unit as that learner, and check that the Learndot enrolment status was set to "PASSED". This will be in the devstack LMS logs, and can also be verified via the Learndot API, using cURL.

**Reviewers**
